assets/js/admin-settings/rule.jsx

```diff
@@ -0,0 +1,145 @@
+/* global wp, wpLocalizedData */
+import {
+  Button,
+  SelectControl,
+  TextControl,
+  ToggleControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import React, { useEffect, useState } from 'react';
+import { useSelect } from '@wordpress/data';
+import { ruleCard } from './styles';
+
+
+const Rule = ({
+  field,
+  loading,
+  newRule,
+  onAdd,
+  onDelete,
+  onUpdate,
+  reorderRule,
+  ruleIndex,
+  saving,
+  setOriginIndex,
+  setTargetIndex,
+  taxonomy,
+  term_id,
+  value,
+}) => {
+  const {
+    fields,
+    taxonomies,
+  } = wpLocalizedData;
+
+  const [rule, setRule] = useState({
+    field: Object.keys(fields)[0],
+    taxonomy: Object.keys(taxonomies)[0],
+    term_id: 0,
+    value: false,
+  });
+
+  // If existing rule, sync local state with incoming settings.
+  // Else set some default values.
+  useEffect(() => {
+    if(!newRule) {
+      setRule({
+        field: field,
+        taxonomy: taxonomy,
+        term_id: term_id,
+        value: value,
+      })
+    } else {
+      setRule({
+        field: Object.keys(fields)[0],
+        taxonomy: Object.keys(taxonomies)[0],
+        term_id: 0,
+        value: false,
```

Comment:
When you set defaults in the main form, set `field`, `taxonomy`, and `value` to `''` and set `term_id` to `0`.